### ISSUE TITLE
Build for ARM platforms like raspberry

### ISSUE THREAD

**Issue by guillaumebadin** [CleverCloud/clever-tools#235]
I try to build clever-tools on ARM rasberry with rasbian. 

First, I dit a npm install.

**Comment by BlackYoup** [comment]
The npm version seems to be quite old: `npm -v 1.4.21`, node 8 is supposed to be packaged with npm ^5. Can you try to update npm and run npm install again?

**Comment by BlackYoup** [comment]
No, it's not. You need curl development headers to have the `<curl/curl.h>` file (usually, the package is named `libcurl-dev` or something like this). You might also want to have openssl (or gnutls) development headers too

**Comment by hsablonniere** [comment]
@guillaumebadin Could you try again with the latest master, we have a need build in place.

**Comment by hsablonniere** [comment]
@guillaumebadin Could you try again with the latest master, we have a new build in place ;-)

**Comment by guillaumebadin** [comment]
Ok, i will try it when I have been on my home.

**Comment by davlgd** [comment]
I checked `clever-tools` (installed through `npm`) on a Raspberry Pi 4 under Ubuntu, it's working fine : 
```
> clever diag
clever-tools  3.0.2
Node.js       v21.1.0
Platform      linux
Release       6.2.0-1016-raspi
Architecture  arm
Linux         Ubuntu 23.04
Shell         /usr/bin/zsh
Packaged      false
Exec path     /usr/bin/node
```

Maybe we could explore ARM support for Linux distros, macOS & Windows with the new packaging stack

**Comment by hsablonniere** [comment]
We'll consider a dedicated build for ARM in the CI pipeline but it's not a big priority for now as the install works via npm.
